tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js

```diff
@@ -0,0 +1,162 @@
+		cy.get( '.nfd-main-heading__title' )
+			.should('be.visible')
+			.contains('website');
+	} );
+
+	it( 'Enter a Title and then Check if it reflects elsewhere', () => {
+		const title = 'Hello WordPress';
+		const titleBox = cy.get( ':nth-child(1) > label > .nfd-input__field' );
+		if ( titleBox.should( 'exist' ) ) {
+			titleBox.scrollIntoView();
+			titleBox.clear();
+			cy.wait( 1000 );
+			titleBox.type( title );
+
+			// Check if Mini Preview Tab has the Title
+			cy.get( '.browser-row-title_bar_main-text' ).contains( title );
+			// Check if Mini Preview Webpage Search result has the Title
+			cy.get( '.browser-content_top-row-name' ).contains( title );
+		}
+	} );
+
+	it( 'Enter a Desc and then Check if it reflects elsewhere', () => {
+		const desc = 'Welcome to WordPress';
+		const descBox = cy.get( ':nth-child(2) > label > .nfd-input__field' );
+		if ( descBox.should( 'exist' ) ) {
+			descBox.scrollIntoView();
+			descBox.clear();
+			cy.wait( 1000 );
+			descBox.type( desc );
+
+			// Check if Mini Preview Webpage Search result has the Desc
+			cy.get( '.browser-content_desc' ).contains( desc );
+		}
+	} );
+
+	it( 'Check if Social Media Accordion Toggles', () => {
+		cy.get(
+			':nth-child(7) > .social-form__label > .social-form__label_name'
+		)
+			.should( 'exist' )
+			.should( 'not.be.visible' );
+
+		// Open Social Media Accordion
+		cy.get( '.social-form__top-row_icon' ).click();
+		cy.get(
+			':nth-child(7) > .social-form__label > .social-form__label_name'
+		)
+			.should( 'exist' )
+			.scrollIntoView()
+			.should('have.css', 'opacity', '1');
+	} );
+
+	it( 'Check if Social Media URL checks are done', () => {
+		const invalidURL = 'htt';
+		const validURL = 'https://www.facebook.com';
+
+		// Facebook Social Media Component
+		const socialTest = cy.get( '#facebook' );
+
+		if ( socialTest.should( 'exist' ) ) {
+			socialTest.clear();
+			cy.get(
+				'.browser-content_social_icon[style="background-image: var(--facebook-icon);"]'
+			).should( 'have.css', 'opacity', '0.5' );
+
+			socialTest.type( invalidURL );
+
+			// The URL Checker runs on a debounce
+			// Shows the message to the User in case of Invalid URL
+			cy.get( '.Tooltip-Wrapper', { timeout: 3000 } ).should( 'exist' );
+			cy.get(
+				'.browser-content_social_icon[style="background-image: var(--facebook-icon);"]'
+			).should( 'have.css', 'opacity', '0.75' );
+
+			socialTest.clear();
+			socialTest.type( validURL );
+			cy.get( '.Tooltip-Wrapper', { timeout: 3000 } ).should(
+				'not.exist'
+			);
+			cy.get(
+				'.browser-content_social_icon[style="background-image: var(--facebook-icon);"]'
+			).should( 'have.css', 'opacity', '1' );
+
+			// Close Social Media Accordion
+			cy.get( '.social-form__top-row_icon' ).click();
+		}
+	} );
+
+	it( 'Check if Image gets Uploaded', () => {
+		const sampleLogo = `vendor/newfold-labs/wp-module-onboarding/tests/cypress/fixtures/${ Cypress.env('pluginId') }-logo.png`;
```

Comment:
Since the site logo can be any generic image and does not have to be a brand specific logo, does it make sense to just name it `logo.png` or something generic along those lines and use a generic image?